### PR TITLE
Add updates for lgtm CI security site

### PR DESCRIPTION
**Pull request by koning**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to Merlin will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Code updates to satisy lgtm CI security checker
```
Comment (lucpeterson) [inline CHANGELOG.md:9]: typo --> satisfy

**Review by lucpeterson — APPROVED** [review]
Ok when we fix the changelog typo.
Question: How did this fix the ssl bug?

File: merlin/config/configfile.py [hunk 2]
```diff
@@ -298,7 +298,8 @@ def merge_sslmap(server_ssl: Dict[str, Union[str, ssl.VerifyMode]], ssl_map: Dic
             new_server_ssl[ssl_map[k]] = server_ssl[k]
         else:
             new_server_ssl[k] = server_ssl[k]
-    server_ssl = new_server_ssl
```
Comment (lucpeterson) [inline merlin/config/configfile.py:301]: was this the big bug?

Comment (koning) [reply merlin/config/configfile.py:301]: Yes

File: CHANGELOG.md (hunk 1 above)
Comment (koning) [reply CHANGELOG.md]: Fixed

**Pull request merged**